### PR TITLE
Add emptydrops

### PR DESCRIPTION
**Pull request by nh3**
This PR adds a new recipe `emptydrops` (https://pypi.org/project/emptydrops/).

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by nh3** [comment]
@BiocondaBot please add label

File: recipes/emptydrops/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  sha256: 16552739852ccb39347f6f1ecddbe10abf4b14ae6b59d5c7b1d4091ad9806cb8
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install --no-deps --ignore-installed -vv ."
+
+requirements:
+  host:
+    - python >=3
```
Comment (bgruening) [inline recipes/emptydrops/meta.yaml:19]: I think you need pip here.

File: recipes/emptydrops/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+test:
+  imports:
+    - emptydrops
+    - emptydrops.matrix
+
+about:
+  home: https://pypi.org/project/emptydrops/
+  dev_url: https://github.com/nh3/emptydrops
+  license: MIT
```
Comment (bgruening) [inline recipes/emptydrops/meta.yaml:34]: can you please add a license_file

Comment (nh3) [reply recipes/emptydrops/meta.yaml:34]: You mean adding this file (https://github.com/nh3/emptydrops/blob/master/LICENSE)? Surely very ignorant, but I haven't seen a sub-directory under `recipe/` having a license_file.

Comment (bgruening) [reply recipes/emptydrops/meta.yaml:34]: Normally you are shipping this file in your pypi package. In this case you just add `license_file: LICENSE`

Comment (nh3) [reply recipes/emptydrops/meta.yaml:35]: Ah, I see. Thanks!

**Comment by nh3** [comment]
@bgruening Now ready for reviewing again.

**Comment by nh3** [comment]
Thank you!

**Comment by nh3** [comment]
@BiocondaBot please fetch artifacts

**Comment by nh3** [comment]
@BiocondaBot please merge

**Pull request merged**